src/core/a-node.js

```diff
@@ -292,6 +292,8 @@ ANode.evtData = {};
 ANode.newMixinIdArray = [];
 ANode.oldMixinIdArray = [];
 ANode.mixinIds = {};
+ANode.DOMContentHasLoaded = false;
```

Comment:
I originally thought of adding this to `window.AFRAME`, but I see that's not a current pattern in the codebase, so added it to the ANode static props instead.